reproserver/web/templates/index.html

```diff
@@ -1,51 +1,117 @@
-{% set current_nav = 'reproduce' %}
+<!-- {% set current_nav = 'reproduce' %} -->
 
 {% extends "base.html" %}
 
 {% block content %}
 
-<h1>How to Use This Site</h1>
-<p>This site allows you to run reproducible research packaged with <a href="https://www.reprozip.org/">ReproZip</a> directly from your browser. Upload your own <code>.rpz</code> file and click the "Upload" button!</p>
-
-<p>From there, you can set parameters and (optionally) upload your own inputs, and then reproduce the work by clicking the "Run" button. You can then see the log and output files from the research, which you can view or download to your machine. You can also share the link for other people to reproduce your research.</p>
-
-<p>You can also choose one of the examples below to test it out!</p>
-
-<h1>Examples</h1>
-
-<p><a href="{{ reverse_url('reproduce_repo', 'osf.io', 'r2ptb') }}">bechdel.rpz</a>: Attempt to replicate the findings from a FiveThirtyEight article examining gender bias in the movie business</p>
-<p><a href="{{ reverse_url('reproduce_repo', 'osf.io', '5ztp2') }}">digits_sklearn_opencv.rpz</a>: Recognizing the value of hand-written digits using OpenCV and scikit-learn</p>
-<p><a href="{{ reverse_url('reproduce_repo', 'osf.io', '5ke97') }}">bash-count.rpz</a>: Simple bash script counting the lines in a file</p>
-
-<h1>Select a package to upload</h1>
-
-<form method="POST" action="{{ reverse_url('upload') }}" enctype="multipart/form-data">
-  {{ xsrf_form_html() }}
-  <div class="mb-3">
-    <label for="rpz_file" class="form-label">Upload a <code>.rpz</code> file</label>
-    <input type="file" class="form-control" id="rpz_file" name="rpz_file" placeholder="experiment.rpz" accept=".rpz">
+<div class="container-fluid shadow p-5 mb-4 bg-light rounded-3">
+  <h1> Share reproducible work </h1>
+  <p>Reprozip allows you to automatically pack your web application along with necessary libraries, configuration
+    files and environment variables</p>
+  <p>Anyone can re run the packed web application on a new machine without tracking and installing the dependencies
+    manually</p>
+  <p>This application is still in progess, Currently It enables you reproduce Reprozip packages from web browser,
+    without having to install reprounzip on your computer</p>
```

Comment:
https://github.com/VIDA-NYU/reproserver/issues/73  - this mentions the new home page, Please correct me if I'm wrong

Reply:
The problem is that web capture is a work in progress but the rest works. Either way I think we were changing to have the web capture option after the upload, so I guess this will be reworked.